R/colour-text.r

```diff
@@ -1,8 +1,9 @@
-
+# fix the check NOTE:Namespace in Imports field not imported from: ‘crayon’
```

Comment:
Yeah, but that's not what the check is for - if that same call was inside a function, `R CMD check` wouldn't complain about it.